ios/Classes/TiMapSnapshotterProxy.m

```diff
@@ -0,0 +1,23 @@
+/**
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2009-2016 by Appcelerator, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+
+#import "TiMapSnapshotterProxy.h"
+#import "TiMapSnapshotter.h"
+
+@implementation TiMapSnapshotterProxy
+
+-(NSString*)apiName
+{
+    return @"Ti.Map.Snapshotter";
+}
+
+-(void)takeSnap:(id)value
+{
+    TiMapSnapshotter *Snapshotter = (TiMapSnapshotter *)[self view];
```

Comment:
Same here. We need to put everything in the proxy (this class) and remove the `TiMapSnapshotter` class.